dotnet/src/webdriver/BiDi/Browser/BrowserModule.cs

```diff
@@ -80,6 +82,24 @@ public async Task<SetDownloadBehaviorResult> SetDownloadBehaviorDeniedAsync(SetD
 
     protected override void Initialize(JsonSerializerOptions options)
     {
-        _jsonContext = new BiDiJsonSerializerContext(options);
+        options.TypeInfoResolverChain.Add(new BiDiJsonSerializerContext());
```

Comment:
When we create `BiDiJsonSerializerContext` instance, should we provide `options` to it?